packages/fxa-components/package.json

```diff
@@ -18,13 +18,20 @@
     "node": ">=12"
   },
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "storybook": "start-storybook -p 6006",
```

Comment:
We _may_ want to have different ports for different storybook instances, especially as we're moving components around into `fxa-components` and want to check them in payments/settings, but I'm also fine with leaving them all at 6006 until someone says it's an inconvenience 🤷‍♀️